react-app-rewired/headers/csps/wallets/keepkey.ts

```diff
@@ -1,5 +1,8 @@
 import type { Csp } from '../../types'
 
 export const csp: Csp = {
-  'connect-src': [process.env.REACT_APP_KEEPKEY_VERSIONS_URL!],
+  'connect-src': [
+    process.env.REACT_APP_KEEPKEY_VERSIONS_URL!,
+    'http://localhost:1646/',
```

Comment:
can this be configurable please, e.g. `REACT_APP_KEEPKEY_DESKTOP_URL`

Reply:
its static, and does not change per environment. but sure